tests/routes_tests/test_jurisdictions.py

```diff
@@ -32,7 +34,11 @@ def jurisdiction_ids(client: FlaskClient, election_id: str) -> List[str]:
     )
     assert json.loads(rv.data) == {"status": "ok"}
     bgcompute_update_election_jurisdictions_file()
-    jurisdictions = Jurisdiction.query.filter_by(election_id=election_id).all()
+    jurisdictions = (
+        Jurisdiction.query.filter_by(election_id=election_id)
+        .order_by(Jurisdiction.name)
```

Comment:
Adding the constraint seemed to change the order of records returned by this query, so I added an `order by` to make it deterministic.

Reply:
Seems like an improvement